src/js/controller/model.ts

```diff
@@ -338,6 +343,7 @@ export type MediaModelAttributes = {
     mediaState: InternalPlayerState;
     position: number;
     preloaded: boolean;
+    seekRange?: SeekRange;
```

Comment:
This doesn't go here. It goes in MediaModelAttributes (added in https://github.com/jwplayer/jwplayer/pull/3702)